LibTest/js_interop/ObjectToJSBoxedDartObject/toJSBox_A01_t02.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A JavaScript object that wraps this [Object].
+///
+/// There are no usable members in the resulting [JSBoxedDartObject] and you may
+/// get a new [JSBoxedDartObject] when calling [toJSBox] on the same Dart
+/// [Object].
+///
+/// Throws an [Exception] if this [Object] is a JavaScript value.
+///
+/// Unlike [ObjectToExternalDartReference.toExternalReference], this returns a
+/// JavaScript value. Therefore, the representation is guaranteed to be
+/// consistent across all platforms and interop members can be declared on
+/// [JSBoxedDartObjects].
+///
+/// @description Checks that an [Exception] is thrown if this [Object] is a
```

Comment:
nit: you're checking that this *doesn't* throw an exception. It currently does on the JS compilers.